dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1264,6 +1291,7 @@ synchronized String evaluateInterval() {
         try {
             queueDepth = threadPool.getQueue().size();
             boolean queueEmpty = (queueDepth <= 0);
+            activeThreads = threadPool.getActiveCount();
```

Comment:
Should this call executorService.getActiveThreads() instead?

Reply:
I am thinking it shouldn't be executorService.getActiveThreads() because it is how many threads are active and are waiting to run.

Reply:
threadPool.getActiveCount() seems to do what I want - when things are busy, activeThreads is equal to poolSize, as the system quiets down activeThreads falls below poolSize.
I'm trying to keep this relatively simple - at this snapshot time, were most of the threads working or were many of them idle.